docs/upgrading/upgrading_to_v0x.md

```diff
@@ -1,11 +1,28 @@
 ---
-id: upgrading-to-v03
-title: Upgrading to v0.3
+id: upgrading-to-v0x
+title: Upgrading to v0.x
```

Comment:
Maybe better this way?

```suggestion
id: upgrading
title: Upgrading guide
```

also we should probably add a redirect, the nginx config now lives in the apify-docs repo

https://github.com/apify/apify-docs/blob/master/nginx.conf#L312